### ISSUE TITLE
Enhancement reduce : Use enumRef:getValue() instead of INTEGER(enumRef) reporting

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#1045]
related to #929 not sure if this should be new issue or comment on the existing

We get loads of 
```
Use enumRef:getValue() instead of INTEGER(enumRef)
```
for Enums with integer values that will never ever be int64 e.g.

```
enum be.cce.ccetools.logging.LoggingLevelEnum :

    define enum
        Fatal = 0
        Error = 1
        Warning = 2
        Information = 3
        Debug = 4
        .

end enum.
```

IMO string(enum) and integer(enum) are more 4GL like constructs than GetValue().

as far as I know I cannot disable this detection without also disabling rssw-oe-main:eu.rssw.antlr.proparse.checks.VariableOverflow

Can this rule detect the highest value of the enum and only warn if it is above integer limits ?
Alternatively can you add an option on VariableOverflow to ignore enums ?

**Comment by gquerret** [comment]
Duplicate of #928 (through #929 is also on a related topic). That's on the roadmap.
